flow_pre_lda.scala

```diff
@@ -134,18 +142,139 @@ def bin_column(value: String, cuts: Array[Double]) = {
 }
 
 
+def convert_feedback_row_to_flow_row(feedBackRow: String) = {
```

Comment:
ohmygodyes

...

but I didn't want to do it in this checkin, seemed like too much to cram into a hotfix kinda situation